packages/react/src/components/UIShell/SwitcherItem.js

```diff
@@ -37,9 +43,28 @@ const SwitcherItem = React.forwardRef(function SwitcherItem(props, ref) {
     [`${prefix}--switcher__item-link--selected`]: isSelected,
   });
 
+  function setTabFocus(evt) {
+    if (match(evt, keys.ArrowDown)) {
```

Comment:
I think we'll want to add this to prevent the screen from scrolling when using the arrow keys to navigate the menu, 
same for `ArrowUp`

```suggestion
    if (match(evt, keys.ArrowDown)) {
       evt.preventDefault();
```